devices/stm32l0x0.yaml

```
@@ -0,0 +1,156 @@
+_svd: ../svd/stm32l0x0.svd
+
+_modify:
+  Flash:
+    name: FLASH
+  SYSCFG_COMP:
+    name: SYSCFG
+
+ADC:
+  CFGR1:
+    _modify:
+      AUTDLY:
+        name: WAIT
+  SMPR:
+    _modify:
+      SMPR:
+        name: SMP
+
+CRC:
+  POL:
+    _modify:
+      Polynomialcoefficients:
+        name: POL
+
+DBG:
+  _modify:
+    DBG_APB2_FZ:
+      name: APB2_FZ
+
+FLASH:
+  _modify:
+    OBR:
+      name: OPTR
+  _delete:
+    - WRPROT1
+    - WRPROT2
+  ACR:
+    _modify:
+      DESAB_BUF:
+        name: DISAB_BUF
+  PECR:
+    _modify:
+      FTDW:
+        name: FIX
+  OPTR:
+    _modify:
+      SPRMOD:
+        name: WPRMOD
+      RDPRT:
+        name: RDPROT
+
+"SPI*,I2S*":
+  SR:
+    _modify:
+      TIFRFE:
+        name: FRE
+
+RCC:
+  CR:
+    _modify:
+      CSSLSEON:
+        name: CSSHSEON
+  CCIPR:
+    _merge:
+      - "LPTIM1SEL*"
+      - "I2C3SEL*"
+      - "I2C1SEL*"
+      - "LPUART1SEL*"
+      - "USART2SEL*"
+      - "USART1SEL*"
+  CSR:
+    _modify:
+      LPWRSTF:
+        name: LPWRRSTF
+    _delete:
+      - LSIIWDGLP
+  AHBSMENR:
+    _modify:
+      CRYPTSMEN:
+        name: CRYPSMEN
+  APB1ENR:
+    _modify:
+      TIM3EN:
+        bitOffset: 1
```

Similarly it seems TIM3 doesn't exist here at all.

```suggestion
    _delete:
      - TIM3EN
```